lib/os/cbprintf_complete.c

```diff
@@ -1165,32 +1169,31 @@ static char *encode_float(double value,
 		}
 	}
 
+	int decimals;
```

Comment:
blank line after declaration?  I usually get diagnostics about this sort of thing, not sure why CI doesn't show one.